<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWordView</class>
 <widget class="QWidget" name="EditWordView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lbWordDesc">
     <property name="text">
      <string>Word:</string>
     </property>
     <property name="buddy">
      <cstring>leWord</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KLineEdit" name="leWord"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbTypeDesc">
     <property name="text">
      <string comment="Grammar category">Category:</string>
     </property>
     <property name="buddy">
      <cstring>cbType</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="KComboBox" name="cbType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbAddTerminal">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbSampa">
     <property name="text">
      <string>Pronunciation:</string>
     </property>
     <property name="buddy">
      <cstring>leSampa</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KLineEdit" name="leSampa"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leWord</tabstop>
  <tabstop>leSampa</tabstop>
  <tabstop>cbType</tabstop>
  <tabstop>tbAddTerminal</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
